articles/primeagen-takes-as-fakes-as-his-mustache/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Primeagen Takes as Fakes as his Mustache"
+date: 2024-12-06
+tags: 
+  - news
+  - tech
+author: "https://x.com/thoughtlesslabs"
+displayName: "thoughtlesslabs"
+---
+
+It turns out ThePrimeagen’s greatest hot take wasn’t about neovim btw or brazil, brazil mentioned, it was perched under his nose. Literally. The internet’s favorite high-energy streamer, known for breaking down developer debates with the enthusiasm of a startup founder pitching a product no one asked for, has been exposed for wearing a fake mustache. Yes, that signature 'stache, once thought to embody wisdom and unshakable coding prowess, was nothing more than a stick-on accessory. A tragic day for facial hair enthusiasts and devs everywhere.
```

Comment:
In the first couple of sentences the reader should realize that his mustache might be fake

When writing a news article it doesn't sound too good to directly jump to a conclusion